gravitee-io/gravitee-access-management
# fix: remove unnecessary audit event

https://gravitee.atlassian.net/browse/AM-599


removed unnecessary REGISTRATION_VERIFY_ACCOUNT_EMAIL_SENT logging event

To test:

Create domain and application
Enable User registration within Login
Enable Account verification via email within User accounts
Initiate login flow
Sign up
AM Console > Audit Log

REGISTRATION_VERIFY_EMAIL_SENT should be triggered only
